animareflection/ui
# Feature/avatar component

## Description

##### Task link: https://trello.com/c/Fvz98TFA/216-avatar-component

Added `Avatar` component

## Test Steps

1. Ensure the component's structure, stories, and testing suite are robust.
2. Confirm that all tests are successful.
3. Check that the Avatar component displays correctly in both storybook and sample downstream applications.
